client/src/components/Navbar/Navbar.tsx

```diff
@@ -90,6 +94,24 @@ const Navbar: React.FC = () => {
               </MenuItem>
             </Menu>
           </>
+        ) : (
+          <>
+            <Box sx={{ display: 'flex', alignItems: 'center' }}>
+              <Link to={'/signup'} className={classes.linkButton}>
+                BECOME A SITTER
```

Comment:
Please avoid using CAPITALCASE text as some screen readers interpret them as abbreviations. You can use a CSS class to transform the text instead :)